charts/portworx/values.yaml

```diff
@@ -89,6 +89,7 @@ kvdb:
 volumes:                              # Add volumes to Portworx container. Supported volume types: Host, Secret, ConfigMap
   # - name: objectstore-cert
   #   mountPath: /etc/pwx/objectstore-cert
+  #   mountPropagation: None|HostToContainer|Bidirectional  
   #   secret:
```

Comment:
@hitesh-wani-px are we missing readOnly field here?

Reply:
Done added now.